certification/internal/shell/has_required_labels_test.go

```diff
@@ -0,0 +1,44 @@
+package shell
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("HasRequiredLabels", func() {
+	var (
+		hasRequiredLabelsCheck HasRequiredLabelsCheck
+		labels                 map[string]string
+	)
+
+	BeforeEach(func() {
+		labels = map[string]string{
+			"name":        "name",
+			"vendor":      "vendor",
+			"version":     "version",
+			"release":     "release",
+			"summary":     "summary",
+			"description": "description",
+		}
+	})
+
+	Describe("Checking for required labels", func() {
+		Context("When it has required labels", func() {
+			It("should succeed the check", func() {
+				ok, err := hasRequiredLabelsCheck.validate(labels)
```

Comment:
Do we plan to test `Validate`, similar to baseOnUbiCheck?

Reply:
This will be in a future push.